packages/css-engine/src/core/style-sheet-regular.ts

```diff
@@ -0,0 +1,28 @@
+import { StyleRule } from "./rules";
+import type { TransformValue } from "./to-value";
+import { StyleSheet, type CssRule } from "./style-sheet";
+
+const defaultMediaRuleId = "__default-media-rule__";
+
+export class StyleSheetRegular extends StyleSheet {
+  addStyleRule(
+    rule: CssRule,
+    selectorText: string,
+    transformValue?: TransformValue
+  ) {
+    const mediaRule = this.addMediaRule(rule.breakpoint || defaultMediaRuleId);
+    if (mediaRule === undefined) {
```

Comment:
Probably better to throw in `addMediaRule` so in ts it would always return defined value

Reply:
done